src/modules/back-to-top/back-to-top.component.scss

```diff
@@ -0,0 +1,31 @@
+.back-to-top {
+    position: fixed;
+    bottom: 10px;
+    right: 10px;
+    color: #FFF;
+    background: #1c84c6;
+    border-radius: 5px;
+    height: 30px;
+    width: 30px;
+    text-align: center;
+    font-size: 2em;
+    line-height: 1;
+    opacity: 0;
+    transition: opacity .5s;
+}
+
+.back-to-top:hover,
+.back-to-top:focus {
+    background: #1a7bb9;
+    cursor: pointer;
+}
+
+i.fa {
+    font-size: medium;
+    display: block;
+    margin-top: 5px;
+}
+
+.show {
```

Comment:
Common classnames for this behavior are `.hidden` or `.visible` (I prefer `.hidden` because an element is visible by default.)